frontend/src/globals/unity-util.ts

```diff
@@ -1094,6 +1094,46 @@ export class UnityUtil {
 		UnityUtil.toUnity('ScaleClipGizmo', UnityUtil.LoadingState.VIEWER_READY, percent);
 	}
 
+	/**
+	 * Sets the scale factor used by the Gizmos that determines how precise the user needs to be to select the axis arrow or arch.
+	 * Default scale is 1.0
+	 * @category Configurations
+	 * @param newScale
+	 */
+	public static setClipGizmoSelectionScale(newScale: number) {
```

Comment:
Isn't this missing in the visualsettings dialog?

Reply:
yes. I don't think we need it (yet) so trying to reduce the amount of settings we expose to avoid confusing them